tree/treeplayer/inc/TTreeReader.h

```diff
@@ -284,7 +284,8 @@ class TTreeReader: public TObject {
    enum EStatusBits {
       kBitIsChain = BIT(14), ///< our tree is a chain
       kBitHaveWarnedAboutEntryListAttachedToTTree = BIT(15), ///< the tree had a TEntryList and we have warned about that
-      kBitSetEntryBaseCallingLoadTree = BIT(16) ///< SetEntryBase is in the process of calling TChain/TTree::%LoadTree.
+      kBitSetEntryBaseCallingLoadTree = BIT(16), ///< SetEntryBase is in the process of calling TChain/TTree::%LoadTree.
+      kBitIsExternalTree = BIT(15)  ///< we do not own the tree
```

Comment:
```suggestion
      kBitIsExternalTree = BIT(17)  ///< we do not own the tree
```
Otherewise `kBitIsExternalTree == kBitHaveWarnedAboutEntryListAttachedToTTree` ?

Reply:
Nice catch: apologies for the mistake. It has been fixed with the recent update of the PR.